mesop/components/markdown/markdown.py

```diff
@@ -28,6 +28,8 @@ def markdown(
     html = markdown_lib.markdown(
       text,
       extensions=[
+        "attr_list",
```

Comment:
Can you add an example of where you set an attribute? specifically, i'm wondering if you set the style attribute whether Angular will sanitize it out (which is OK, but then I'm not sure how helpful it is to have an attribute list here).

Reply:
Good point. I tested it out just now. Seems to work. Updated the code with an example.

Reply:
I guess there's no way to do inline style (e.g. `style="background: red"`)? If users set CSS class names, it's not very straight forward to define CSS class rules (I think you would need to [inject a stylesheet](https://google.github.io/mesop/api/page/#mesop.features.page.page:~:text=DEFAULT%3A%20None-,stylesheets,-List%20of%20stylesheet))